5.analyze-data/README.md

```diff
@@ -0,0 +1,34 @@
+# 5. Analyze Data
+
+In this module, we perform the analysis of model-generated data.
+
+### Model Probability Analysis
+
+We use the model phenotype probabilities generated from [non-shuffled](https://github.com/WayScience/phenotypic_profiling_model/blob/main/2.train_model/models/multi_class_models/final__CP_areashape_only__balanced.joblib) and [shuffled](https://github.com/WayScience/phenotypic_profiling_model/blob/main/2.train_model/models/multi_class_models/shuffled_baseline__CP__balanced.joblib) weighted logistic regression models, trained exclusively from [mitocheck](https://github.com/WayScience/mitocheck_data) cellprofiler areashape morphology features.
```

Comment:
Maybe explain why we use areashape morphology feature?

Reply:
Added